.goreleaser.yml

```diff
@@ -41,123 +44,123 @@ dockers:
     goos: linux
     goarch: amd64
     ids:
-    - listmonk
+      - listmonk
```

Comment:
Really not much has changed here. It's all just got better through the variable usage. And because that was properly formatted etc it looks like everything changed but barely anything changed.